Workiva/over_react_codemod
# FED-2094 Add suggestor that replaces useRef with useRefInit

## Motivation
  

## Changes
  

#### Release Notes
  

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
        - [ ] Run the following commands on any of the repos in [this SG query](https://sourcegraph.plat.workiva.net/search?q=context:global+lang:dart++-repo:react_material_ui+-repo:react-dart+-repo:over_react+useRef%28%3F:%3C%5B%5E%28%5D%2B%3E%29%3F%5C%28%5B%5E%5C%29%5D&patternType=regexp&sm=1&groupBy=repo) locally
        ```
        dart pub global activate https://github.com/Workiva/over_react_codemod.git --source=git --git-ref=batch/fedx/FED-2094_useRefInit_migration
        dart pub global run over_react_codemod:use_ref_init_migration --yes-to-all
        ```
        and verify that the `useRef` usages that pass an initial value have been changed to `useRefInit`

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:


[contributing-review-types]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#manual-testing-criteria
